loculus_values/values.yaml

```diff
@@ -910,7 +932,7 @@ defaultOrganismConfig: &defaultOrganismConfig
       - name: referenceGenomeAccession
```

Comment:
I think this field should be removed as it will potentially differ from the reference sequence we use and will be confusing